jetbrains-gateway/README.md

```diff
@@ -72,6 +72,22 @@ module "jetbrains_gateway" {
 }
 ```
 
+### Custom base link
+
+```tf
+module "jetbrains_gateway" {
+  source                = "registry.coder.com/modules/jetbrains-gateway/coder"
+  version               = "1.0.21"
+  agent_id              = coder_agent.example.id
+  agent_name            = "example"
+  folder                = "/home/coder/example"
+  jetbrains_ides        = ["GO", "WS"]
+  releases_base_link    = "https://releases.internal.site/"
+  download_base_link    = "https://download.internal.site/"
+  default               = "GO"
+}
+```
+
```

Comment:
Thank you. This example is helpful. But can you also add on how one can set up internal serving for JetBrains IDEs? Thanks